src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -467,6 +467,17 @@ public ResponseEntity<String> getNetworkElementInfos(
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getNetworkElementInfos(studyUuid, nodeUuid, elementType, infoType, elementId, operation, inUpstreamBuiltParentNode));
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-map/countries")
+    @Operation(summary = "Get network countries")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The list of countries")})
+    public ResponseEntity<String> getNetworkCountries(
+            @Parameter(description = "Study uuid") @PathVariable("studyUuid") UUID studyUuid,
+            @Parameter(description = "Node uuid") @PathVariable("nodeUuid") UUID nodeUuid,
+            @Parameter(description = "Should get in upstream built node ?") @RequestParam(value = "inUpstreamBuiltParentNode", required = false, defaultValue = "false") boolean inUpstreamBuiltParentNode) {
```

Comment:
Add inUpstreamBuiltParentNode=true

Reply:
Corrected in request from front-end
https://github.com/gridsuite/gridstudy-app/pull/1658/commits/fef4bab75d83dc7ef8bd57d92ba9343d123b6c51#diff-a05894790b3ecf0b35dce0445a1317e2172b6739be46d88b524702c5085fe60bR136